decidim-bulletin_board-app/spec/cypress/integration/sandbox/election.page.js

```diff
@@ -0,0 +1,222 @@
+              {
+                force: true,
+              }
+            );
+
+          cy.findByText("Start").click();
+          cy.findByText("Backup").click();
+        });
+    });
+  }
+
+  /**
+   * Assert that the key ceremony has ended checking the election status.
+   *
+   * @param {Object} election - An Election object.
+   * @param {Array<Object>} trustees - A collection of Trustee objects.
+   *
+   * @returns {undefined}
+   */
+  assertKeyCeremonyHasEnded(election, trustees) {
+    trustees.forEach(({ name }) => {
+      cy.contains(name)
+        .parent("tr")
+        .within(() => {
+          cy.findByText("Done").should("be.visible");
+        });
+    });
+
+    cy.findByText("Back").click();
+
+    this.withinElectionRow(election, () => {
+      cy.findByText("key_ceremony").should("not.exist");
+      cy.findByText("key_ceremony_ended").should("be.visible");
+    });
+  }
+
+  /**
+   * Starts the vote process for the given election.
+   *
+   * @param {Object} election - An Election object.
+   *
+   * @returns {undefined}
+   */
+  startVote(election) {
+    this.withinElectionRow(election, () => {
+      cy.findByText("Start vote").click().should("not.exist");
+    });
+  }
+
+  /**
+   * Assert that the vote has started checking the election status.
+   *
+   * @param {Object} election - An Election instance.
+   *
+   * @returns {undefined}
+   */
+  assertVoteHasStarted(election) {
+    this.withinElectionRow(election, () => {
+      cy.findByText("key_ceremony_ended").should("not.exist");
+      cy.findByText("vote").should("be.visible");
+    });
+  }
+
+  /**
+   * Cast a vote for the given election.
+   *
+   * @param {Object} election - An Election instance.
+   *
+   * @returns {undefined}
+   */
+  castVote(election) {
+    this.withinElectionRow(election, () => {
+      cy.findByText("Vote").click();
+    });
+    cy.findByText("Cast vote").should("not.be.disabled").click();
+  }
+
+  /**
+   * Assert that the vote has been casted successfully.
+   *
+   * @returns {undefined}
+   */
+  assertVoteHasBeenCasted() {
+    cy.findByText("Your vote has been casted successfully").should(
+      "be.visible"
+    );
+    cy.findByText("Back").click();
+  }
+
+  /**
+   * End the vote process for the given election.
+   *
+   * @param {Object} election - An Election object.
+   *
+   * @returns {undefined}
+   */
+  endVote(election) {
+    this.withinElectionRow(election, () => {
+      cy.findByText("End vote").click().should("not.exist");
+    });
+  }
+
+  /**
+   * Assert that the vote has ended checking the election status.
+   *
+   * @param {Object} election - An Election instance.
+   *
+   * @returns {undefined}
+   */
+  assertVoteHasEnded(election) {
+    this.withinElectionRow(election, () => {
+      cy.findByText("vote").should("not.exist");
+      cy.findByText("vote_ended").should("be.visible");
+    });
+  }
+
+  /**
+   * Prepares a context to find elements in a election row from the list.
```

Comment:
```suggestion
   * Prepare a context to find elements in an election row from the list.
```
Here I just removed the s so it's consistent with the rest of the comments.

Reply:
> Here I just removed the s so it's consistent with the rest of the comments.

I'm going to fix it in #95 👍 , thanks!